docs/contributing/features.md

```diff
@@ -42,6 +42,7 @@ If there's agreement that the feature belongs in one or more of the core stacks:
    If you use `make`, call:
 
    ```bash
+   OWNER={yourdockerhubusername}
    make build/somestack-notebook
```

Comment:
This won't work, because make will still use `OWNER?=jupyter` line instead of a variable you set.

Reply:
Ah yeah I'll need to update the owner refs (per https://stackoverflow.com/questions/24263291/define-a-makefile-variable-using-a-env-variable-or-a-default-value ) as well.

Reply:
We probably won't need that after dealing with local registry as suggested above.